app/mikane/src/app/pages/account/danger-zone/danger-zone.component.html

```diff
@@ -0,0 +1,36 @@
+
+<mat-card *ngIf="(breakpointService.isMobile() | async) === false; else mobileView">
+	<mat-card-header>
+		<div mat-card-avatar>
+			<mat-icon aria-hidden="false">key</mat-icon>
+		</div>
+		<mat-card-title>Danger zone</mat-card-title>
+	</mat-card-header>
+	<mat-card-content>
+		<ng-container *ngTemplateOutlet="dangerZoneContent"></ng-container>
+	</mat-card-content>
+</mat-card>
+
+<ng-template #mobileView>
+	<div class="title-mobile">Danger zone</div>
+	<div class="danger-zone-mobile">
+		<ng-container *ngTemplateOutlet="dangerZoneContent"></ng-container>
+	</div>
+</ng-template>
+
+<ng-template #dangerZoneContent>
+  <div>
+    <h2>Delete your account</h2>
+  </div>
+  <div class="delete-account-text">
+    When the deletion process is initiated by clicking the button below you will get a confirmation email with the deletion link. Think hard before you press this link!
+    <br><br>
+    If you decide to proceed, you should know: once an account has been deleted you cannot go back. There is no undo button. The data is gone permanently and no one can get it back for you.
+  </div>
+  <mat-card-actions>
+    <button type="button" mat-stroked-button color="warn" (click)="deleteUser()">
```

Comment:
Maybe don't allow button to be clicked if getting user fails. Same thing in edit user component. Maybe show an error message instead of regular content

Reply:
Good point - added as part of new issue #198